fs/procfs/fs_procfsmeminfo.c

```diff
@@ -314,6 +317,17 @@ static ssize_t meminfo_read(FAR struct file *filep, FAR char *buffer,
           buffer    += copysize;
           buflen    -= copysize;
 
+#ifdef CONFIG_BUILD_PROTECTED
+          /* Trigger free_delaylist operation once */
+
+          ptr = mm_malloc(entry->heap, 0);
```

Comment:
merge line 283 here

Reply:
Why not change `free_delaylist()` to public function?